### PR TITLE
Add missing README

### PR DESCRIPTION
**Pull request by heavyimage**
Gotta have a README, bro

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# README
```
Comment (arcadeperfect) [inline README.md:1]: Add a more descriptive title please

File: README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
```
Comment (arcadeperfect) [inline README.md:2]: The license and a brief description of the project would usually go here.

It doesn't need to be that long, three to four hundred words is probably enough.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+## Running the code
+"just do `cargo run --release`"
```
Comment (arcadeperfect) [inline README.md:4]: It wouldn't hurt to outline the reasons for the --release flag

**Comment by arcadeperfect** [comment]
This needs work, see comments

**Comment by heavyimage** [comment]
I will not.

**Comment by heavyimage** [comment]
I hope future employers won't see this and think I'm some kind of horrid code jerk instead of a good friend trying to help a buddy out in a non-passive aggressive and delightful way.

**Comment by arcadeperfect** [comment]
And they probably will, see this i mean

File: README.md (hunk 3 above)
Comment (heavyimage) [reply README.md:4]: this is a direct copy/paste from your message to me so if anyone is to blame here, it's you.

**Pull request merged**